src/chip.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {createUseStyles} from './jss';
+import {useTheme} from './hooks';
+import Box from './box';
+import {Text2} from './text';
+import IconButton from './icon-button';
+import IconCloseRegular from './generated/mistica-icons/icon-close-regular';
+
+const useStyles = createUseStyles(({colors}) => ({
+    container: {
+        display: 'inline-flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        border: `1px solid ${colors.border}`,
+        borderRadius: 16,
```

Comment:
Use 20px because in desktop with a bigger text the rounded is not 100% rounded (the difference is very little but exists :D)

Reply:
changed